### PR TITLE
Theme setup

### PR DESCRIPTION
**Pull request by yasminnvaz**
# Description

Please include a summary of the changes and features that were added. Also include which JIRA task is this related to (name and link). List any dependencies that were added or removed for this change.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings or errors;
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] My commits follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) Guidelines;

### CODE REVIEW

File: packages/react/.storybook/preview-head.html [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<link href="https://fonts.googleapis.com/css2?family=Roboto+Mono&family=Roboto:wght@400;500&display=swap" rel="stylesheet" />
+<link href="./natds-icons.css" rel="stylesheet"/>
```
Comment (arielwb) [inline packages/react/.storybook/preview-head.html:2]: pra usar o natds-icons assim tu tem que incluir um script que copie a fonte pra tua pasta build do storybook.
eu recomendo usar a url do cnd e evitar essa volta 
https://unpkg.com/@naturacosmeticos/natds-icons@latest/dist/natds-icons.css

File: packages/react/package.json [hunk 2]
```diff
@@ -42,16 +42,21 @@
...
+    "lint": "eslint --ext .js,.jsx,.ts,.tsx ./",
     "lint:fix": "yarn lint --fix",
     "rollup:build": "rollup -c",
     "storybook": "start-storybook -p 6006",
     "test": "jest --verbose",
+    "test:ci": "jest --coverage",
     "test:watch": "jest --watch"
   },
+  "dependencies": {
+    "@naturacosmeticos/natds-themes": "0.4.12",
```
Comment (arielwb) [inline packages/react/package.json:57]: sugiro nao fixar a dependencia do natds-themes pra usuarias pegarem updates, so deixar o ^ no inicio

File: packages/react/src/components/Button/Button.stories.tsx [hunk 3]
```diff
@@ -1,34 +1,56 @@
...
-
-export const Large = Template.bind({})
-Large.args = {
-  size: 'large',
-  label: 'Button'
-}
+export const Sizes: Story<ButtonProps> = (args) => (
+  <>
+    <Button {...args} label="semi" size="semi" />
+    &nbsp;&nbsp;
```
Comment (arielwb) [inline packages/react/src/components/Button/Button.stories.tsx:45]: nao rola usar css pra dispor os elementos e matar os `&nbsp;&nbsp;`?

File: packages/react/src/ThemeProvider/ThemeProvider.tsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { ThemeProvider as Provider, JssProvider } from 'react-jss'
+import { ThemeProviderProps } from './ThemeProvider.props'
+import buildTheme from './buildTheme'
+
+const ThemeProvider = ({
+  children, brand, mode, cssPrefix = 'natds'
```
Comment (arielwb) [inline packages/react/src/ThemeProvider/ThemeProvider.tsx:7]: acho que o prefixo padrão poderia ser `natds-` pq sem o `-` as classes estão sendo geradas assim `natdsbutton`

File: packages/react/package.json [hunk 5]
```diff
@@ -42,16 +42,21 @@
   "scripts": {
     "build": "yarn clean && tsc -p ./tsconfig.json && yarn rollup:build",
     "build:storybook": "yarn build-storybook -o docs/dist",
-    "build-storybook-ci": "yarn build:storybook --quiet",
+    "build:storybook:ci": "yarn build:storybook --quiet",
     "clean": "rm -rf ./dist",
     "copy:assets": "mkdir -p ./dist/assets && cp -r ./src/assets/* ./dist/assets",
-    "lint": "eslint --ext .js,.jsx,.ts,.tsx  ./",
+    "lint": "eslint --ext .js,.jsx,.ts,.tsx ./",
```
Comment (arielwb) [inline packages/react/package.json:48]: eu nao consegui ver como ta o teu `eslint ignore` mas confere se com essa config (./) não ta pegando tbm as pastas de dist, coverage etc.
Acho que se tu rodar o lint com --verbose ele imprime o path dos arquivos, pode te ajudar a ver se tem algo que nao precisa sendo lintado

File: packages/react/.storybook/preview-head.html (hunk 1 above)
Comment (yasminnvaz) [reply packages/react/.storybook/preview-head.html:2]: Feito!

File: packages/react/package.json (hunk 2 above)
Comment (yasminnvaz) [reply packages/react/package.json:57]: Feito!

File: packages/react/src/ThemeProvider/ThemeProvider.tsx (hunk 4 above)
Comment (yasminnvaz) [reply packages/react/src/ThemeProvider/ThemeProvider.tsx:7]: Feito!

File: packages/react/package.json (hunk 5 above)
Comment (yasminnvaz) [reply packages/react/package.json:48]: hmm, testei aqui e estava funcionando sim. O que fiz agora foi executar via lerna para cada pacote, silenciando `warnings` de natds-styles e natds-web

File: packages/react/src/components/Button/Button.stories.tsx (hunk 3 above)
Comment (yasminnvaz) [reply packages/react/src/components/Button/Button.stories.tsx:45]: concordo!

**Pull request merged**